packages/xrpl/src/models/transactions/trustSet.ts

```diff
@@ -89,6 +93,10 @@ export interface TrustSetFlagsInterface extends GlobalFlags {
   tfSetFreeze?: boolean
   /** Unfreeze the trust line. */
   tfClearFreeze?: boolean
+  /** Deep-Freeze the trust line -- disallow sending and recieving the said IssuedCurrency */
+  tfSetDeepFreeze?: boolean
```

Comment:
ditto

Reply:
fixed in [c9ecd33](https://github.com/XRPLF/xrpl.js/pull/2873/commits/c9ecd33eb88848d25d1dca2a24c5270ac356cb04)